augur/data/schema-auspice-config-v2.json

```diff
@@ -181,7 +184,13 @@
                     "type": "string",
                     "pattern": "^(none|[a-zA-Z0-9]+)$"
                 },
+                "tip_label": {
```

Comment:
The inclusion of `tip_label` is a bug - as Auspice doesn't yet support this tip label. Unless nextstrain/augur#1692 gets merged soon we should revert this from the schema.
I'm sorry I missed this in review, I'll remember to look more carefully at schema changes.
@jameshadfield 

I've made an issue for better support of different auspice schema versions here: https://github.com/nextstrain/augur/issues/1326